Source/WebKit/WPEPlatform/wpe/wayland/WPEWaylandSeat.cpp

```diff
@@ -671,6 +671,16 @@ void WaylandSeat::handleKeyEvent(uint32_t time, uint32_t key, uint32_t state, bo
             return;
         }
         break;
+    case WPE_KEY_F11:
+        if (!state || eventModifiers)
+            break;
+        auto* toplevel = WPE_TOPLEVEL(m_keyboard.toplevel.get());
+        auto topLevelState = wpe_toplevel_get_state(toplevel);
+        if (topLevelState & WPE_TOPLEVEL_STATE_FULLSCREEN)
+            wpe_toplevel_unfullscreen(toplevel);
+        else
+            wpe_toplevel_fullscreen(toplevel);
+        break;
```

Comment:
I don't think this should be handled here. Keybindings should be handled by browser applications, like MiniBrowser does. Not all apps might want the same key shortcuts.